cogs/quotes.py

```diff
@@ -1,17 +1,19 @@
-import random
-
 from discord.ext import commands
 
+
 class Quotes(commands.Cog):
     def __init__(self, bot):
         self.bot = bot
 
     @commands.group(invoke_without_command=True)
     async def quote(self, ctx, search=""):
+        """Lists one quote that use an optional query"""
```

Comment:
The optional query is already indicated by the help syntax. A simple "Lists a random quote" should be sufficient.